src/utils/dataJSON/helper/helper.ts

```
@@ -88,3 +95,30 @@ export function generateUUID(): string {
+  // documents.push("docs", "domicileCertificate");
+
+  function formatKey(key: any) {
+    const spacedKey = key.replace(/([a-z])([A-Z])/g, "$1 $2");
+
+    const normalizedKey = spacedKey.replace(/_/g, " ");
+
+    return normalizedKey.replace(/\b\w/g, (char: any) => char.toUpperCase());
+  }
+
+  for (const key in applicationData) {
+    if (applicationData.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)